### DISCUSSION TITLE
Add TokenizingTextBox to Settingscard or SettingsCard Item?!

### DISCUSSION THREAD

**Discussion by TimoB90** [CommunityToolkit/Windows#439] [Q&A]
Hello togehter,

is there a way to combine TokenizingTextBox with SettingsCard? If I will do that, my application crash instantly.

This is what I tried in my XAML file:

```
xmlns:labs="using:CommunityToolkit.Labs.WinUI"
xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"

 <labs:SettingsCard Header="Title">
     <labs:SettingsCard.HeaderIcon>
         <FontIcon Glyph="&#xE97C;" />
     </labs:SettingsCard.HeaderIcon>
     <controls:TokenizingTextBox  PlaceholderText="Add Tags" />
 </labs:SettingsCard>
```

If the TokenizingTextBox is added to my SettingsCard I get an HResult E_FAIL

![Unbenannt](https://github.com/CommunityToolkit/WindowsCommunityToolkit/assets/86852558/097c31a9-38e9-44a8-9888-61acbc4ef787)

Is that not possible? If I remove the TokenizingTextBox, the page is loading.

Thanks for your help.

**Comment by TimoB90** [comment]
If I add  a width to the TokenizingTextBox everything works

**Comment by niels9001** [comment]
@TimoB90 

I see that you are still using the Labs version of the `SettingsControls`. With 8.0 we have moved `SettingsControls` into the main Toolkit.

Could you try to replace Nuget package with this one: https://www.nuget.org/packages/CommunityToolkit.WinUI.Controls.SettingsControls

**Reply by TimoB90** [reply]
Checked it, same behavior. As long as I don't set a width for TokenizingTextBox within a Settingscard the my tool crash.

Do you have the same conspicuity?

**Reply by niels9001** [reply]
I wonder if this is a TokenizingTextBox issue vs a SettingsCard issue? See: https://github.com/CommunityToolkit/Windows/issues/297
